radix-transactions/src/builder/transaction_builder.rs

```diff
@@ -140,8 +140,16 @@ impl TransactionV1Builder {
     }
 }
 
+#[derive(Default)]
 pub struct PartialTransactionV2Builder {
```

Comment:
Maybe it was already discussed...
Any reasons for using `Partial` instead of `Sub`?

Reply:
Ok, had second thought on it. 
Probably because this is not a subtransaction but rather a part of transaction :)
Ignore it

Reply:
Right - exactly.  This is effectively a "subtree" of a transaction. But I think I was worried that calling it a SubTransaction might be confusing. That said, maybe it's better. Will raise it in the original naming thread as your suggestion :).